releasenotes/notes/add-primitives-support-31af39549b5e66e3.yaml

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
Could we do some prelude section here that has a high level summary - i.e. more or less that Qiskit Optimization has been updated to support the primitive based algorithms introduced in Qiskit Terra 0.22. The older algorithms that took or used quantum instance, are still supported here until they are removed from Qiskit Terra in a future release. We encourage you to migrate to using the new primitive based algorithms.

Or something like that which just gives a high level statement. Will we do migration tutorial that shows examples of how it would have been done and how its done now with primitive ones. Since the optimization code is the same I think its really down to the algorithm creation I believe. It maybe helpful to show this anyway in a couple of examples. Maybe that could be added in the PR with the tutorial updates if it seems useful?

Reply:
I made a small suggestion but I think its good and captures what is new in the release as a high level overview for the reader.